src/dialog/Input.tsx

```diff
@@ -0,0 +1,91 @@
+/** @jsx JSXSlack.h */
+import { JSXSlack } from '../jsx'
+import { ObjectOutput } from '../utils'
+import { Text, TextProps } from './Text'
+
+interface InputPropsBase {
+  children?: undefined
+  hint?: string
```

Comment:
There is an idea to improve compatibility with HTML attributes by aliasing `title` prop to `hint`.